src/containers/Experiment/SamplesTable.scss

```diff
@@ -215,7 +215,37 @@
   }
 }
 
+.info-icon {
+  vertical-align: sub;
+}
+
+.metadata-modal {
+  padding: 56px 24px 56px 56px;
+  max-width: 108 * 8px;
+  overflow: hidden;
+
+  &__title {
+    margin-bottom: 4px;
+  }
 
+  &__subtitle {
+    margin-top: 4px;
+  }
+
+  &__annotations {
+    overflow: auto;
+    height: 700px;
+    border-radius: 2px;
+    color: #fdfdfd;
+    font-family: 'Oxygen Mono';
+  }
+
+  pre {
+    padding: 8px;
+    white-space: pre-wrap;
+    background-color: #4a4a4a;
```

Comment:
What about putting this `background-color` into the `annotations` element above? So that the black square stays in the same place when scrolling.

![2018-08-22 12 34 31](https://user-images.githubusercontent.com/1882507/44477117-c5e68180-a607-11e8-9218-7d0d56a3655b.gif)

Reply:
Great point! I have made the change. Thanks.